### PR TITLE
fix(deps): move commitlint packages to devDependencies

### PR DESCRIPTION
**Pull request by ybiquitous**
`commitlint` ~~and `chai`~~ packages seem to be used only on development.

- `commitlint`: used on pre commit
- ~~`chai`: used on test~~

This PR removes these extra dependencies from runtime dependencies.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -78,6 +76,8 @@
     "@types/rimraf": "2.0.2",
     "@types/sinon": "4.3.1",
     "@types/temp": "0.8.32",
+    "chai": "^4.0.1",
+    "chai-as-promised": "^7.0.0",
     "commitizen": "^2.9.6",
```
Comment (ybiquitous) [inline package.json]: [note] It looks `commitizen` should be removed... (_unrelated to this PR_)

**Comment by ybiquitous** [comment]
Thanks for your review! I rewrote the commit, the PR title & message.

**Comment by ybiquitous** [comment]
I have a question. Is there no need to include also `mocha` & `sinon` into `dependencies`?

**Comment by felixfbecker** [comment]
I guess yes

**Comment by ybiquitous** [comment]
Thank you!

**Pull request merged**